src/wallet/rpcwallet.cpp

```diff
@@ -1070,7 +1070,10 @@ UniValue addmultisigaddress(const UniValue& params, bool fHelp)
             "       \"address\"  (string) bitcoin address or hex-encoded public key\n"
             "       ...,\n"
             "     ]\n"
-            "3. \"account\"      (string, optional) DEPRECATED. An account to assign the addresses to.\n"
+            "3. options        (object, optional)\n"
+            "   {\n"
+            "     \"cltv_height\"  (numeric, optional) Minimum block height before received funds can be spent\n"
```

Comment:
This is more complex. SelectCoins will need to be taught not to mix height and time locked coins, and FundTransaction to use time rather than height (preferably in more than merely cases that require it, for privacy). Note the code in this RPC call specifically blocks cltv_time for now.